pkg/action/hooks.go

```diff
@@ -44,7 +43,7 @@ func (cfg *Configuration) execHook(rl *release.Release, hook release.HookEvent,
 
 	for _, h := range executingHooks {
 		// Set default delete policy to before-hook-creation
-		if h.DeletePolicies == nil || len(h.DeletePolicies) == 0 {
```

Comment:
I suspect a linter (correctly) pointed out this is redundant. But IMHO lets keep this PR focused on fixing `errors` related changes only.

Reply:
Ok, will revert for this and others